lib/shuffle.js

```diff
@@ -23,7 +23,12 @@ function nextBytes(numBytes) {
         return randomBytes(numBytes);
     }
     catch (ex) {
-        throw new Error('Insufficient entropy');
+        if (ex instanceof Error) {
+            throw new Error(`Insufficient entropy: ${ex.message}`);
+        }
+        else {
+            throw new Error('Insufficient entropy: Unknown error');
+        }
```

Comment:
```suggestion
        throw new Error(`Insufficient entropy: ${ex.message || "Unknown error"}`);
```

Reply:
this will return an error => 'ex' is of type 'unknown as we need to check is ex is an instanceOf Error before trying to log the message